backend/api/Areas/Reports/Controllers/LeaseController.cs

```diff
@@ -0,0 +1,113 @@
+    [ApiController]
+    [Area("reports")]
+    [ApiVersion("1.0")]
+    [Route("v{version:apiVersion}/[area]/leases")]
+    [Route("[area]/leases")]
+    public class LeaseController : ControllerBase
+    {
+        #region Variables
+        private readonly IPimsService _pimsService;
+        private readonly IMapper _mapper;
+        #endregion
+
+        #region Constructors
+        /// <summary>
+        /// Creates a new instance of a ReportController class, initializes it with the specified arguments.
+        /// </summary>
+        /// <param name="pimsService"></param>
+        /// <param name="mapper"></param>
+        public LeaseController(IPimsService pimsService, IMapper mapper)
+        {
+            _pimsService = pimsService;
+            _mapper = mapper;
+        }
+        #endregion
+
+        #region Endpoints
+        #region Export Leases
+        /// <summary>
+        /// Exports leases as CSV or Excel file.
+        /// Include 'Accept' header to request the appropriate export -
+        ///     ["text/csv", "application/application/vnd.ms-excel", "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet"]
+        /// </summary>
+        /// <param name="all"></param>
+        /// <returns></returns>
+        [HttpGet]
+        [HasPermission(Permissions.LeaseView)]
+        [Produces(ContentTypes.CONTENT_TYPE_CSV, ContentTypes.CONTENT_TYPE_EXCELX)]
+        [ProducesResponseType(200)]
+        [SwaggerOperation(Tags = new[] { "lease", "report" })]
+        public IActionResult ExportLeases(bool all = false)
+        {
+            var uri = new Uri(this.Request.GetDisplayUrl());
+            var query = Microsoft.AspNetCore.WebUtilities.QueryHelpers.ParseQuery(uri.Query);
+            return ExportLeases(new Lease.Models.Search.LeaseFilterModel(query), all);
+        }
+
+        /// <summary>
+        /// Exports leases as CSV or Excel file.
+        /// Include 'Accept' header to request the appropriate export -
+        ///     ["text/csv", "application/application/vnd.ms-excel", "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet"]
+        /// </summary>
+        /// <param name="filter"></param>
+        /// <param name="all"></param>
+        /// <returns></returns>
+        [HttpPost("filter")]
+        [HasPermission(Permissions.LeaseView)]
+        [Produces(ContentTypes.CONTENT_TYPE_CSV, ContentTypes.CONTENT_TYPE_EXCELX)]
+        [ProducesResponseType(200)]
+        [SwaggerOperation(Tags = new[] { "lease", "report" })]
+        public IActionResult ExportLeases([FromBody] Lease.Models.Search.LeaseFilterModel filter, bool all = false)
+        {
+            filter.ThrowBadRequestIfNull($"The request must include a filter.");
+            if (!filter.IsValid()) throw new BadRequestException("Lease filter must contain valid values.");
+            var accept = (string)this.Request.Headers["Accept"] ?? throw new BadRequestException($"HTTP request header 'Accept' is required.");
```

Comment:
nit: accept => acceptHeader

Reply:
This would be tech debt as this is a pattern that we use across the application.

Reply:
https://jira.th.gov.bc.ca/browse/PSP-2499